Gemfile

```diff
@@ -6,4 +6,5 @@ gem 'jruby-openssl', :platforms => :jruby
 group :test, :remote_test do
   # gateway-specific dependencies, keeping these gems out of the gemspec
   gem 'braintree', '>= 2.0.0'
+  # gem 'braintree', github: "braintree/braintree_ruby", branch: "raw_apple_pay_preview" # for apple pay support in braintree
```

Comment:
actually (as we discussed offline), going to wait to merge this until we have the ability to enable apple pay for a merchant ID through their API as well.